src/renderer/components/ft-prompt/ft-prompt.js

```diff
@@ -30,14 +30,21 @@ export default Vue.extend({
     }
   },
   mounted: function () {
-    document.querySelector('.prompt button').focus({ focusVisible: true })
+    this.focusItem(0)
   },
   methods: {
     removeWhitespace,
+    hide: function() {
+      this.$emit('click', null)
+    },
     handleHide: function (event) {
       if (event.target.getAttribute('role') === 'button' || event.target.className === 'prompt') {
-        this.$emit('click', null)
+        this.hide()
       }
+    },
+    focusItem: function (index) {
+      document.querySelector(`#prompt-${removeWhitespace(this.label)}-${index}`)
```

Comment:
This seems to be broken for the restart prompt for the smooth scrolling toggle:
![smooth_scrolling_prompt_error](https://user-images.githubusercontent.com/48293849/195392957-3738f7ee-9335-4a06-b2bd-550b39c89dfd.jpg)

Reply:
I'll update the replace function to remove special characters as well

Reply:
Might be easier to use a regex whitelist e.g. `/^[a-Za-z0-9]$/`, that way you don't have to worry about adding more characters to the blacklist in the future.